Plugin~/Src/MeshSync/Include/MeshSync/SceneGraph/msInstanceInfo.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include "MeshUtils/muMath.h"
+#include "MeshSync/msFoundation.h"
+
+namespace ms {
+	using namespace std;
+	using namespace mu;
+	class InstanceInfo
+	{
+		uint64_t m_hash = 0;
```

Comment:
not sure what would be a good and efficient hash for this.. maybe a random number when the transforms member is assigned?